pymc3/distributions/discrete.py

```diff
@@ -793,6 +793,53 @@ def _repr_latex_(self, name=None, dist=None):
                                                 get_variable_name(p))
 
 class HyperGeometeric(Discrete):
+	 R"""
```

Comment:
Bad indent.

Reply:
Corrected that in the subsequent commit.